cfme/tests/containers/test_relationships.py

```diff
@@ -66,15 +78,15 @@ def test_relationships_tables(provider, has_persistent_volume, appliance, test_i
     if text.isdigit():
         view = appliance.browser.create_view(test_item.obj.all_view)
         value = int(text)
-        items_amount = int(view.paginator.items_amount)
```

Comment:
I forgot to update items_amount for 5.9 when developers added appropriate JS enhancement. 
My fix was merged yetsterday. So, I believe you can use items_amount here

Reply:
Yes!!!